### PR TITLE
Remainder of Ingress additions, deploy/[subdirectory] support for config files

### PR DESCRIPTION
**Pull request by emalloy**
Specific changes:

* IGRESSES->INGRESSES
* shell parameter expansion to account and chop subdirs from SERVICE name
* Blocking job source file change: *.blockjob.yml -> *.blockingjob.yml

### CODE REVIEW

**Comment by emalloy** [comment]
@sairez When you get a chance please review, I hope my rebase kept intact the correct changes.

File: k8s-scripts.config [hunk 1]
```diff
@@ -37,3 +37,7 @@ JOBS=()
 
 # List of files ending in '.blockingjob.yml' in the kube directory
 BLOCKING_JOBS=()
+
+# List of files ending in '.deployment.yml' in the kube directory
+DEPLOYMENTS=('example-app')
```
Comment (sairez) [inline k8s-scripts.config]: hmm DEPLOYMENTS is in here twice for some reason

File: bin/verify-deployment [hunk 2]
```diff
@@ -21,8 +21,8 @@ UPDATED_REPLIACS=-1
 REPLICAS=0
 until [ "$UPDATED_REPLIACS" == "$REPLICAS" ]; do
   sleep 1
-  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas | sed -n 2p | awk '{print $2}')
-  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas | awk '{print $2}')
+  REPLICAS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep replicas: | sed -n 2p | awk '{print $2}')
+  UPDATED_REPLIACS=$(kubectl get deployment ${1} --namespace=${2:=default} -o yaml | grep updatedReplicas: | awk '{print $2}')
```
Comment (sairez) [inline bin/verify-deployment]: @philipcristiano can you give me your thoughts on this change? this was added when I had a deployment with `replicas` in the name and the grep got stuck on the wrong value.

Comment (philipcristiano) [inline bin/verify-deployment]: If it works then it looks good. Almost always a fan of more specificity when grepping.

**Pull request merged**